solana/program_v2/packages/client/core/package.json

```diff
@@ -21,16 +20,16 @@
     "node": ">=14"
   },
   "scripts": {
-    "lint": "eslint src/**/*.ts",
-    "build": "tsc -p tsconfig.json"
+    "build": "tsc -p tsconfig.json",
+    "prepack": "yarn install && yarn build"
   },
   "dependencies": {
-    "@identity.com/gateway-solana-idl": "^2.0.0-beta.8",
+    "@identity.com/gateway-solana-idl": "^2.0.0-beta.12",
+    "@solana/web3.js": "^1.50.1",
     "@project-serum/anchor": "^0.26.0",
     "@project-serum/anchor-cli": "^0.26.0"
```

Comment:
I actually don't see why that needs a `anchor-cli` and if it does it should only be a `devDependency`. please remove

Reply:
(I understand that this was not changed in the PR, but needs to be corrected. :))